scripts/install-python.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -x
+
+# Install Python
+yum -y install deltarpm epel-release
+yum -y install python34
```

Comment:
Checked with @jbouffard 3.3 is the min requirement, so this should be ok for now.